example-tests/src/test/java/com/epam/restassured/pageobjects/ThankYouPageObject.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.restassured.pageobjects;
+
+import com.epam.restassured.util.Driver;
+
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import com.google.common.base.Verify;
+
+/**
+ * Created by Peter_Olah1 on 12/10/2015.
+ */
+public class ThankYouPageObject {
```

Comment:
A PageObject, in general, should mimic a page's behaviour. So I do not prefer that validations are conducted by the PageObject.